website/content/en/docs/add-ons/load-balancer/guide.md

```diff
@@ -113,6 +129,9 @@ Set up resources required for the Load Balancer controller:
     export LBC_POLICY_ARN=$(aws iam create-policy --policy-name $LBC_POLICY_NAME --policy-document file://awsconfigs/infra_configs/iam_alb_ingress_policy.json --output text --query 'Policy.Arn')
     eksctl create iamserviceaccount --name aws-load-balancer-controller --namespace kube-system --cluster ${CLUSTER_NAME} --region ${CLUSTER_REGION} --attach-policy-arn ${LBC_POLICY_ARN} --override-existing-serviceaccounts --approve
     ```
+
+#### 3. Configure the Load Balancer Controller
```

Comment:
We will need to split the below section `Build Manifests and deploy components` .

One part will go into the installing LBC controller (terraform users need to skip this step)
```
while ! kustomize build awsconfigs/common/aws-alb-ingress-controller/base | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 30; done
```


and new title for Build Manifests and deploy components can be `Create Ingress`

```
kustomize build awsconfigs/common/istio-ingress/overlays/https | kubectl apply -f -
```

Reply:
done